ExtractCopyright/Properties/AssemblyInfo.cs

```diff
@@ -8,11 +8,6 @@
 [assembly: AssemblyTitle("ExtractCopyright")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("SIL International")]
-[assembly: AssemblyProduct("ExtractCopyright")]
-[assembly: AssemblyCopyright("Copyright © 2017 SIL International")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
```

Comment:
This information is included from the globalassemblies file in the root directory because it applies to all projects in libpalaso. Technically it is a different change than the strong naming change. Here again, I am doing this for consistency with the way other projects in libpalaso handle these 4 attributes.